aws/eks/README.md

```diff
@@ -0,0 +1,112 @@
+# teraform-eks
+This is an opinionated way of getting setting up an EKS cluster using terraform. This module installs several helm packages
```

Comment:
> way of getting setting up an EKS cluster using terraform

Just fix up that grammar a little ;)

Reply:
Fixed in f0a986b32e7373e5ff155849df70242b5b6be470